webapp-django/crashstats/authentication/management/commands/makesuperuser.py

```diff
@@ -30,10 +37,24 @@ def handle(self, **options):
                 )
                 user.set_unusable_password()
 
-            if user.is_superuser:
-                print('{} was already a superuser'.format(user.email))
+            # Set superuser and staff flags on user
+            if user.is_superuser and user.is_staff:
+                self.stdout.write('{} was already a superuser/staff'.format(user.email))
             else:
                 user.is_superuser = True
                 user.is_staff = True
+                self.stdout.write('{} is now a superuser/staff'.format(user.email))
                 user.save()
-                print('{} is now a superuser'.format(user.email))
+
+            # Add user to Hackers group
+            try:
+                hackers_group = Group.objects.get(name='Hackers')
+                if user in hackers_group.user_set.all():
+                    self.stdout.write('{} is already in Hackers group.'.format(user.email))
+                else:
+                    hackers_group.user_set.add(user)
+                    self.stdout.write('{} added to Hackers group.'.format(user.email))
+                    user.save()
+
```

Comment:
Not sure if you meant to put that empty line before the `except` clause.

Reply:
Oops--I can remove this.